dev_guide/deployments.adoc

```diff
@@ -109,6 +109,30 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 <5> The link:#rolling-strategy[rolling strategy] is the default and may be omitted.
 ====
 
+== Deployment Client Tool Operations
+Use the 'ocs deploy' command with available options to start, view, cancel, or
+retry a deployment.
+
+----
+$ oc deploy <deployment_config> [--options]
+----
+
+[cols= 2]
+|===
+|`oc deploy <deployment_config> --latest`
+|Starts the specified deployment
+
+|`oc deploy <deployment_config>`
+|Dsiplays the specified deployment
+
+|`oc deploy <deployment_config> --retry`
+|Attempts to retry to deploy the last deployment. The deployer pod and any hook pods are deleted.
+
+|`oc deploy <deployment_config> --cancel`
+|Cancels the deployment that is currently in progress
```

Comment:
Not sure if this will be the right place to add the information (will this be the canonical docs for the command?) but we might want to point out that the cancelation is a *request* and the deployment will be cancelled *eventually* using a best-effort attempt; it's possible the deployment will be partially or even entirely completed depending on when it's cancelled and when the system tears it all down.